pyincore/analyses/mlenabledcgeslc/mlcgeslc.py

```diff
@@ -145,7 +145,7 @@ def get_spec(self):
             ]
         }
 
-    def run_analysis(self) -> None:
+    def run(self) -> None:
```

Comment:
This doesn't need the "-> None":  If you see the base class, it should return "True" at the end.